### PR TITLE
Feature/log in flow

### PR DESCRIPTION
**Pull request by AndriiStasiuk**

### CODE REVIEW

File: SummerProject/static/src/components/LogIn/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        }
+    }
+
+    changeHandler = event => {
+        this.setState({[event.target.name]: event.target.value})
+    };
+
+    submitHandler = event => {
+        event.preventDefault();
+        console.log(this.state);
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/LogIn/index.js]: Please remove `console.log`

File: SummerProject/static/src/components/Registration/index.js [hunk 2]
```diff
@@ -1,53 +1,144 @@
...
+        };
+    }
+
+    changeHandler = event => {
+        this.setState({[event.target.name]: event.target.value})
+    };
+
+    submitHandler = event => {
+        event.preventDefault();
+        console.log(this.state);
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/Registration/index.js]: Remove `console.log`

File: SummerProject/static/src/components/Registration/index.js [hunk 3]
```diff
@@ -1,53 +1,144 @@
...
+                            </div>
                         </div>
-                    </div>
-                    <div className="passw-email">
-                        <label>Email</label>
-                        <div className="form-group">
-                            <input type="text" placeholder="johndoe@gmail.com" name="email"
-                                   className="form-control input-height"/>
+                        <div className="sign-up-w">
+                            <button type="submit" onClick={this.submitHandler} className="btn-primary sing-up">SIGN UP
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/Registration/index.js:129]: You have two triggers for `submitHandler` (one here and one on line `88`). You only need one, so please remove one of them

File: SummerProject/static/src/components/styles/mainLogInFlow.css [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 33px;
+    color: #000000;
+}
+
+.right-part p {
+    font-family: sans-serif;
+    font-style: normal;
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/styles/mainLogInFlow.css]: Do you ever need `font-family` other than `sans-serif` and `font-style` other than `normal `? You're duplicating these style rules pretty often. You could set these rules for some parent element and all it's child would share them

File: SummerProject/static/src/components/styles/mainLogInFlow.css [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+    cursor: pointer;
+    background-color: #D72130;
+    border: 1px solid #D72130;
+    border-radius: 0;
+    box-sizing: border-box;
+}
+
+
+.btn-primary:hover, .btn-primary:active {
+    background: white !important;
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/styles/mainLogInFlow.css:144]: Are you sure that you need `!important` here?

**Review by UlianaShevtsiv — APPROVED** [review]
+1 after few changes are done

File: SummerProject/static/src/components/styles/mainLogInFlow.css (hunk 5 above)
Comment (AndriiStasiuk) [reply SummerProject/static/src/components/styles/mainLogInFlow.css:144]: Yeah, without !important it doesn't work correctly

File: SummerProject/static/src/components/Registration/index.js (hunk 3 above)
Comment (AndriiStasiuk) [reply SummerProject/static/src/components/Registration/index.js:129]: Done

**Comment by filipenm** [comment]
+1

File: SummerProject/static/src/components/styles/mainLogInFlow.css (hunk 5 above)
Comment (UlianaShevtsiv) [reply SummerProject/static/src/components/styles/mainLogInFlow.css:144]: Ok, just wanted make sure that you really need it.

**Review by AlexK0210 — APPROVED** [review]

**Review by MaksymMartyniak — APPROVED** [review]

**Pull request merged**